cmd/bridgec/erc20cmd.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/threefoldfoundation/tfchain/cmd/bridgec/internal"
+	"github.com/threefoldfoundation/tfchain/pkg/api"
+	"github.com/threefoldtech/rivine/pkg/cli"
+)
+
+// createERC20Cmd creates rootcommand for ERC20 and adds a subcommand
+// if rootcommand executed the user will also see the output of the syncing status of ethereum
+func createERC20Cmd(client *internal.CommandLineClient) *cobra.Command {
+	erc20SubCmds := &erc20SubCmds{cli: client}
+
+	// define Rootcommand
+	var (
+		rootCmd = &cobra.Command{
+			Use:   "erc20",
+			Short: "Perform erc20 actions",
+			Long:  "Perform erc20 actions",
+			Run:   erc20SubCmds.getSyncingStatus,
+		}
+		getSyncingStatusCmd = &cobra.Command{
+			Use:   "syncstatus",
+			Short: "Get the ethereum sync status",
```

Comment:
```suggestion
			Short: "Get the ethereum downloader sync status",
```

And remove the Long description, it doesn't serve a purpose here, as you are not adding any new information. When the long description is not defined, the system will use the short as long as well.